src/main/java/com/openshift/restclient/ClientBuilder.java

```diff
@@ -199,6 +203,13 @@ public IClient build() {
 
 			OkHttpClient.Builder builder = new OkHttpClient.Builder()
 				.addInterceptor(responseCodeInterceptor)
+				.addNetworkInterceptor(new Interceptor() {
+                    @Override
+                    public Response intercept(Chain chain) throws IOException {
+                        Request agent = chain.request().newBuilder().header("User-Agent", "openshift-restclient-java/"+Version.userAgent()).build();
```

Comment:
Yeah I was in a quick / dirty frame of mind when first crafting this, but the pressure on this has decreased a bit.  I still want to include some notion of restclient/fabric version, but I'll change so the ClientBuilder consumer can prepend how they are.

Thanks.